docs/citation.md

```diff
@@ -16,7 +16,7 @@ Please consider contacting the project contact (also available on the project pa
 
 ## Citing the ScPCA Portal
 
-The date you accessed the data from the ScPCA Portal can be found in your data download: once you uncompress your download, the top-most directory name will contain the access date.
+The date you accessed the data from the ScPCA Portal can be found in your data download: once you uncompress your download, the root directory name will contain the access date.
```

Comment:
This thought just occurred to me. How would someone cite the portal if they downloaded data from us over the course of a week and published results from all of the data? 
Do we need to clarify how they would need to cite the portal then? I'm assuming they are not expected to cite the portal with each of the accessed dates??